website/src/examples/layers.mdx

```diff
@@ -0,0 +1,5 @@
+# Dynamic Styling
+
+import App from 'website-examples/layers/src/app';
```

Comment:
nit: rename this example folder to `styling` so its connected to the title?

Reply:
We often link to the examples in issues/discussions and renaming will break them.